src/durableerror.ts

```diff
@@ -0,0 +1,5 @@
+export class DurableError extends Error {
```

Comment:
What is the purpose of this new error class? I suggest we add a comment about why it exists since it doesn't actually add any state or functionality to the base `Error` class.

Reply:
It was just to differentiate errors from the "Durable" framework (i.e. a scheduled activity or suborchestration) failed vs any other exception. Theoretically, customers can then make their catch logic more specific by doing an `instanceof` check. If you don't think it's worth adding, I am fine removing it.

Reply:
I’m fine with keeping it. I just think it would be worth adding a comment in the code about it.